source/content/guides/professional-services/04-advanced-global-cdn.md

```diff
@@ -105,6 +105,24 @@ Support for blue/green deployment<Popover title="Blue/Green Deployment" content=
 
 [See a comparison of the features](https://pantheon.io/product/advanced-global-cdn#pricing-matrix-wrapper) offered by our CDN services.
 
+### Rate Limiting (Early Access)
+
+Rate Limiting, a feature of Advanced Global CDN, lets you place limits on request volume at the network perimeter. Organizations of all sizes can adopt this edge configuration to block malicious traffic. Rate Limiting can be configured to guard critical assets including login, form, and promotional pages by adding another layer of security to your site.
+
+The benefits of Rate Limiting include:
+
+- Volumetric Attack Mitigation - Reduce the effectiveness of malicious traffic, including brute-force login and denial-of-service attacks.
+
+- Policy-based Control - Rate Limiting supports adjustable configurations that can be adjusted on a per policy basis, including requests per second, and detection window.
```

Comment:
"can be adjusted on a per policy basis"

I'd say this is "kind of" true. We can create policies that include configurable RPS and detection windows, but this makes it sound like a customer can have multiple policies within their service. We're only giving them a single policy.